docs/webhooks.md

```diff
@@ -8,47 +8,40 @@ The param `is_base64` used in function `validate` from module `spylib.utils.hmac
 to better convey the meaning of the parameter
 
 
+**Breaking Changes in version 0.7.0**: <br>
+The webhook features become available from `spylib.webhook` module as functions 
+instead of from the `Token` class as methods. 
+
 ## Register Webhooks
 
 You can easily register your endpoint to receive webhooks from Shopify using an admin API access token:
 ```python
-from spylib import OfflineTokenABC
-
-class OfflineToken(OfflineTokenABC):
-    """Example offline token"""
-    async def save(self):
-        # Write to storage
-        pass
-
-    async def load(cls, store_name: str):
-        # API access scopes from https://shopify.dev/api/usage/access-scopes
-        return cls(store_name=store_name, scope=['write_orders'], access_token='ACCESS_TOKEN')
+from spylib import webhook
```

Comment:
Does this work in python? Or is it `from spylib.webhook import webhook`?

Reply:
Yea, I think so.
```py
from spylib import webhook
```
`webhook` here works as a module and the objects inside the `webhook` module will be accessed via `webhook.` e.g. `webhook.WebhookTopic`